cmd/config_options.go

```diff
@@ -28,7 +29,8 @@ type ConfigOptions struct {
 func NewConfigOptions() *ConfigOptions {
 	o := &ConfigOptions{}
 	if err := defaults.Set(o); err != nil {
-		logger.WithError(err).WithField("options", "ConfigOptions").Fatal("error setting driverkit options defaults")
+		slog.With("err", err.Error(), "options", "ConfigOptions").Error("error setting driverkit options defaults")
```

Comment:
`slog` package has no "Fatal" level.
Emulate it through `os.Exit(1)` (that is exactly what Logrus.Fatal did).